README.md

```diff
@@ -1,3 +1,14 @@
+# Checklist for model change #
```

Comment:
Did you want to explicitly list the messages with the `oneof` identified object types to  have the new classes added?

Reply:
is that explicit enough for your liking?

Reply:
Yep that looks good to me.